### ISSUE TITLE
Worldmap export / DeletedApiProjectMapError

### ISSUE THREAD

**Issue by elfrinjo** [subsurface/subsurface#457]
Subsurface Version: 4.6.4-1 (Arch AUR Build)
Browser: Chromium 59.0.3071.109 (and others)

Chromium show the following Java Script errors:

```
Google Maps API error: DeletedApiProjectMapError https://developers.google.com/maps/documentation/javascript/error-messages#deleted-api-project-map-error
_.rb @ js:35

```

**Comment by mturkia** [comment]
Our API key has apparently been deleted by Google. I have no idea why this has been done, but a valid API key is needed for the map to function. Individual users should be able to create their own API key and update the exported worldmap with that, but I am not sure what the licensing terms are for open source projects.

IIRC we concluded that this kind of use is OK when the feature was implemented, but as our key has been deleted, someone should re-check the licensing terms before we get a new API key. Or we should consider options for the user to insert their own key when exporting the worldmap.
